tests/cpp/test_multidevice_communications.cpp

```diff
@@ -26,24 +26,24 @@ class CommunicationTest
 
   void validate(at::Tensor obtained, at::Tensor expected);
 
-  static constexpr DeviceIdxType root = 0;
-  static constexpr int tensor_size = 1024;
+  static constexpr DeviceIdxType kRoot = 0;
+  static constexpr int kTensorSize = 1024;
```

Comment:
The rest of the codebase doesn't consistently follow this leaking-k naming style either (some files do but most files don't). I chose to make these changes anyway because they make the code easier to read by distinguishing local variables and constant variables.